test/Interpreter/generic_casts.swift

```diff
@@ -158,16 +158,6 @@ print(allToAll(type(of: C()), AnyObject.self)) // CHECK-NEXT: true
 // Bridging
 print(allToAll(0, AnyObject.self)) // CHECK-NEXT: true
 
-// This will get bridged using _SwiftValue.
```

Comment:
What's the behavioral change here? It looks like it applies to macOS as well?

Reply:
My bad; this is an artifact of development where _SwiftValue was not yet available. It will be restored as-is.

Reply:
My bad; this is an artifact of development where _SwiftValue was not yet available. It will be restored as-is.